spec/index.bs

```diff
@@ -4894,18 +4899,31 @@ must be well nested.
 <dl dfn-type=method dfn-for=GPUCommandEncoder>
     : <dfn>writeTimestamp(querySet, queryIndex)</dfn>
     ::
+        Writes a timestamp value into |querySet| when all previous commands have completed executing,
+        and sets the query state to `true` at |queryIndex| in the sequence of
+        {{GPUCommandEncoder}}.{{GPUCommandEncoder/[[queries_state_map]]}}.
```

Comment:
nit: This is just part of the validation and not important to the high-level description of what writeTimestamp does. I think it should be moved inside the algorithm, as a step after the validation.